src/libraries/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -21,14 +21,8 @@
     <Compile Include="$(CommonPath)System\Net\InternalException.cs"
              Link="Common\System\Net\InternalException.cs" />
     <!-- System.Net common -->
-    <Compile Include="$(CommonPath)System\Net\Sockets\ProtocolType.cs"
-             Link="Common\System\Net\Sockets\ProtocolType.cs" />
-    <Compile Include="$(CommonPath)System\Net\Sockets\SocketType.cs"
-             Link="Common\System\Net\Sockets\SocketType.cs" />
```

Comment:
We can do it later if you'd prefer, but these files should also move out of common into the appropriate library folder

Reply:
yes, I was thinking about `SocketAddress.cs`  as well once the changes are over. Probably separate PR with just moves/renames.